tools/workbench/src/main/java/org/eclipse/rdf4j/workbench/util/QueryStorage.java

```diff
@@ -229,6 +230,9 @@ public boolean askExists(final HTTPRepository repository, final String queryName
 		LOGGER.info("{}", ask);
 		try (RepositoryConnection connection = this.queries.getConnection()) {
 			return connection.prepareBooleanQuery(QueryLanguage.SPARQL, ask.toString()).evaluate();
+		} catch (Throwable t) {
+			LOGGER.error(t.getMessage());
+			throw t;
```

Comment:
Why does this method in particular need to log Throwable (eg. expections and also things like OOM and assertion errors)?

Reply:
It doesn't, necessarily, this is a leftover from some debugging I did. I'll take it back out.